packages/angular/projects/clr-angular/src/data/datagrid/providers/selection.spec.ts

```diff
@@ -193,7 +193,7 @@ export default function (): void {
         expect(currentSelection.sort(numberSort)).toEqual(itemsInstance.displayed);
         selectionInstance.toggleAll();
         expect(currentSelection).toEqual([]);
-        expect(nbChanges).toBe(3);
+        expect(nbChanges).toBe(4);
```

Comment:
nit: Merging back this PR, I noticed this line and realized I have no idea where the extra change comes from. Any suggestions?

Reply:
Found it. Two `emitChanges()` calls - one in setSelected and one more in updateCurrent(), which is chain-called by setSelected.
Will fix it in the v3/v2 merge directly and then make a separate PR for v4/master

Reply:
done